pkg/steps/pod.go

```diff
@@ -198,3 +178,67 @@ func PodStep(name string, config PodStepConfiguration, resources api.ResourceCon
 		jobSpec:     jobSpec,
 	}
 }
+
+func (s *podStep) generatePodForStep(image string, containerResources coreapi.ResourceRequirements) (*coreapi.Pod, error) {
```

Comment:
here you return `error` but it will always be `nil` since you are not checking for any errors inside this function.